cmd/harmony/flags.go

```diff
@@ -955,6 +959,30 @@ var (
 		Usage:    "listen address for pprof",
 		DefValue: defaultConfig.Pprof.ListenAddr,
 	}
+	pprofFolderFlag = cli.StringFlag{
+		Name:     "pprof.folder",
+		Usage:    "folder to put pprof profiles",
+		DefValue: defaultConfig.Pprof.Folder,
+		Hidden:   true,
+	}
+	pprofProfileNamesFlag = cli.StringSliceFlag{
+		Name:     "pprof.profile.names",
+		Usage:    "a list of pprof profile names (separated by ,) e.g. cpu,heap,goroutine",
+		DefValue: defaultConfig.Pprof.ProfileNames,
+		Hidden:   true,
+	}
+	pprofProfileIntervalFlag = cli.IntSliceFlag{
+		Name:     "pprof.profile.intervals",
+		Usage:    "a list of pprof profile interval integer values (separated by ,)",
+		DefValue: defaultConfig.Pprof.ProfileIntervals,
+		Hidden:   true,
+	}
+	pprofProfileDebugFlag = cli.IntSliceFlag{
+		Name:     "pprof.profile.debug",
+		Usage:    "a list of pprof profile debug integer values (separated by ,)",
```

Comment:
What does each integer mean?

Reply:
0 writes the gzip-compressed protocol buffer. 1 writes the legacy text format with comments translating addresses to function names and line numbers, so that no tools are needed to read the profile. But predefined profiles may assign meaning to other debug values. E.g. for the "goroutine" profile 2 means to print the goroutine stacks in the same form that a Go program uses when dying due to an unrecovered panic. More information can be found here: https://golang.org/pkg/runtime/pprof/

https://github.com/harmony-one/harmony/blob/92568eae89d5cf706b51150ff64e828599d7ed10/cmd/harmony/flags.go#L987-L989

Reply:
This is good description here. Can you add the link to the help message? The E.g. seems to be not very clear.

Also, do you think it is better to have the flag have a default value as 0?